test/e2e/backup_test.go

```diff
@@ -0,0 +1,83 @@
+package e2e
+
+import (
+	"flag"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"golang.org/x/net/context"
+
+	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+)
+
+var (
+	backupName  string
+	restoreName string
+)
+
+var _ = Describe("Backup", func() {
```

Comment:
This test can be moved into platform-specific test files.
Preserving it as it has examples of using `kibishii`